src/app/beer_garden/app.py

```diff
@@ -257,10 +288,10 @@ def _shutdown(self):
             self.scheduler.shutdown(wait=False)
 
         self.logger.debug("Stopping local plugin process monitoring")
-        PluginManager.instance().stop()
+        beer_garden.local_plugins.manager.lpm_proxy.stop()
 
         self.logger.debug("Stopping local plugins")
-        PluginManager.instance().stop_all()
+        beer_garden.local_plugins.manager.lpm_proxy.stop_all()
```

Comment:
Again, only this class references lpm_proxy, so it feels more like only this class should use it.

Reply:
That's going to change